app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,8 @@
+class FoodsController < ApplicationController
+  def index
+    if params[:q].present?
```

Comment:
sad path/edge case handling wasn't an expectation for this assessment, but it's always good to see those handled 👍🏻